src/main/java/com/glco/user/service/UserServiceImpl.java

```diff
@@ -18,11 +18,21 @@ public class UserServiceImpl implements UserService {
     @Override
     @Transactional
     public UserSignupResponseDto signup(UserSignupRequestDto requestDto) {
+        validateDuplicateEmail(requestDto); //중복 이메일 검증
+        validateDuplicateNickname(requestDto); //중복 닉네임 검증
```

Comment:
각 함수에서 사용하는건 각각 email, nickname 뿐인데 DTO를 다 넘기는건 의미가 없지 않을까요?
```suggestion
        validateDuplicateEmail(requestDto.getEmail());
        validateDuplicateNickname(requestDto.getNickname());
```